CoverMyDotNet.Tests/ClientTests.cs

```diff
@@ -204,6 +205,233 @@ public void Should_Client_Delete_Post_Correct_Data()
 				Assert.GreaterOrEqual(requestContent.IndexOf("Steve"), 0);
 			}	
 		}
-	}
-}
 
+		[Test]
+		public void Should_Client_Search_Drug_Use_Correct_Data()
+		{
+			string drug_name = Guid.NewGuid().ToString();
```

Comment:
I think I moved removed this test from this branch and added it to the form and drug search PR.